### PR TITLE
Fix #34: Don't neglect to call create_jinja_environment()

### LINKED ISSUE #34: Fix: Flask.jinja_env should return self.create_jinja_environment()
Hint: See #33 if you get stuck

### PR DESCRIPTION
**Pull request by jab**
plz merge

### CODE REVIEW

File: flask/app.py [hunk 1]
```diff
@@ -674,7 +674,7 @@ def jinja_env(self):
         accessed. Changing :attr:`jinja_options` after that will have no
         effect.
         """
-        return None
+        return self.create_jinja_environment()
```
Comment (mayuropensource) [inline flask/app.py]: Remove trailing white space.

**Review by mayuropensource — CHANGES_REQUESTED** [review]
please make requested changes.

File: flask/app.py (hunk 1 above)
Comment (jab) [reply flask/app.py]: Oops, sorry about that, fixing now

Comment (mayuropensource) [reply flask/app.py]: awesome! thanks!

**Pull request merged**